taskcluster/translations_taskgraph/actions/rebuild_toolchains.py

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from taskgraph.actions.registry import register_callback_action
+from taskgraph.actions.util import create_tasks, fetch_graph_and_labels
+
+
+@register_callback_action(
+    name="rebuild-toolchains",
+    title="Rebuild Toolchains",
+    symbol="rebuild-toolchains",
+    description="Create toolchain tasks to rebuild their artifacts.",
+    order=1000,
+    context=[],
+)
+def rebuild_toolchains_action(parameters, graph_config, input, task_group_id, task_id):
+    decision_task_id, full_task_graph, label_to_taskid = fetch_graph_and_labels(
+        parameters, graph_config, task_group_id=task_group_id
+    )
+    toolchains = [label for label, task in full_task_graph.tasks.items() if task.kind == "toolchain"]
```

Comment:
I think we should rename this action and include docker images and fetches here as well, as they'll cause all the same problems as toolchains do.

Reply:
Mm, any ideas on names? We can create those kinds too.

Reply:
How about "Rebuild docker images and toolchains". (I don't think "fetch" needs to be included in the names, seeing as they just feed toolchains.)